components/tools/OmeroPy/test/unit/clitest/test_web.py

```diff
@@ -328,30 +341,47 @@ def testApacheWSGIConfig(self, prefix, http, capsys, monkeypatch):
         assert not missing, 'Line not found: ' + str(missing)
 
     @pytest.mark.parametrize('server_type', [
-        "nginx", "nginx-wsgi", "nginx-development",
-        "apache", "apache-fcgi", "apache-wsgi"])
+        ["nginx", 'fastcgi-tcp'],
+        ["nginx-wsgi", 'wsgi-tcp'],
+        ["nginx-development", 'fastcgi-tcp'],
+        ["nginx-wsgi-development", 'wsgi-tcp'],
+        ["apache", 'fastcgi-tcp'],
+        ["apache-fcgi", 'fastcgi-tcp'],
+        ["apache-wsgi", 'wsgi']])
     def testFullTemplateDefaults(self, server_type, capsys, monkeypatch):
-        self.args += ["config", server_type]
+        app_server = server_type[-1]
+        del server_type[-1]
```

Comment:
Sorry I wasn't sure how to add additional params, or use some sort of namespace?